### ISSUE TITLE
Template only glimmer components

### ISSUE THREAD

**Issue by chbonser** [ember-learn/ember-cli-addon-docs#763]
Is addon docs able to document template only glimmer components? If so, are there any good examples?

**Comment by rwwagner90** [comment]
I'm not sure actually. I think most of the generated docs typically come from JS, but you could add your own `md` files to manually document your components as well.

**Comment by chbonser** [comment]
Yes I can use the `md` files to manually document them, however that leaves a discrepancy between the auto generated component section and my manually documented section which is potentially confusing the users of the docs.

**Comment by rwwagner90** [comment]
@chbonser ah yes, that would be less than ideal. Is this something you have time to investigate or perhaps help with implementing? I'm just trying to keep this repo afloat, but have not had a lot of time to do a deep dive into it.

**Comment by ijlee2** [comment]
@rwwagner90 I haven't dived into the repo before, but can try to provide support as well.
